packages/tpetra/core/src/Tpetra_Map_decl.hpp

```diff
@@ -1574,30 +1574,28 @@ namespace Tpetra {
     return Teuchos::rcp (new out_map_type (cloner_type::clone (*this, nodeOut)));
   }
 
-  namespace Details {
-    /// \brief Is map1 locally fitted to map2?
-    ///
-    /// \param map1 [in] The first Map
-    /// \param map2 [in] The second Map
-    ///
-    /// For Map instances map1 and map2, we say that map1 is
-    /// <i>locally fitted</i> to map2 (on the calling process), when
-    /// the initial indices of map1 (on the calling process) are the
-    /// same and in the same order as those of map2.  "Fittedness" is
-    /// entirely a local (per MPI process) property.
-    ///
-    /// The predicate "is map1 fitted to map2 ?" is <i>not</i>
-    /// symmetric.  For example, map2 may have more entries than map1.
-    ///
-    /// Fittedness on a process can let Tpetra avoid deep copies in
-    /// some Export or Import (communication) operations.  Tpetra
-    /// could use this, for example, in optimizing its sparse
-    /// matrix-vector multiply.
-    template <class LocalOrdinal,class GlobalOrdinal, class Node>
-    bool
-    isLocallyFitted (const Tpetra::Map<LocalOrdinal, GlobalOrdinal, Node>& map1,
-                     const Tpetra::Map<LocalOrdinal, GlobalOrdinal, Node>& map2);
-  } // namespace Details
+  /// \brief Is map1 locally fitted to map2?
+  ///
+  /// \param map1 [in] The first Map
+  /// \param map2 [in] The second Map
+  ///
+  /// For Map instances map1 and map2, we say that map1 is <i>locally
+  /// fitted</i> to map2 (on the calling process), when the indices of map1
+  /// (on the calling process) are the same and in the same order as the
+  /// initial indices of map2.  "Fittedness" is entirely a local (per MPI
+  /// process) property.
+  ///
+  /// The predicate "is map1 fitted to map2 ?" is <i>not</i>
+  /// symmetric.  For example, map2 may have more entries than map1.
+  ///
+  /// Fittedness on a process can let Tpetra avoid deep copies in
+  /// some Export or Import (communication) operations.  Tpetra
+  /// could use this, for example, in optimizing its sparse
+  /// matrix-vector multiply.
+  template <class LocalOrdinal,class GlobalOrdinal, class Node>
+  bool
+  isLocallyFitted (const Tpetra::Map<LocalOrdinal, GlobalOrdinal, Node>& map1,
```

Comment:
See above: I'm OK with promoting this to the public interface of Tpetra, as long as you like the name.